### PR TITLE
IOGT 6: Section image layout

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review 

RTL
screenshot 2016-08-08 10 45 21

LRT
screenshot 2016-08-08 10 45 36

File: iogt/static/css/state/_s-article.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (peterp) [inline iogt/static/css/state/_s-article.scss:1]: @Mitso Is this for the feature phone or smart phone styles? If it's for smart phone then perhaps it should be included in the `@media` query part.

**Comment by Mitso** [comment]
@peterp that's correct, thanks. This is updated

**Comment by peterp** [comment]
👍

**Pull request merged**